src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -161,6 +188,13 @@ public void drive(double xSpeed, double ySpeed, double rot, boolean fieldRelativ
         setModuleStates(swerveModuleStates);
   }
 
+  public void driveRobotRelative(ChassisSpeeds robotRelativeSpeeds) {
+    ChassisSpeeds targetSpeeds = ChassisSpeeds.discretize(robotRelativeSpeeds, 0.02);
```

Comment:
What does `discretize()` do? Is 0.02 something we might want to tune? If so, it might be worth making this a named constant.

Reply:
`discretize` adjusts `ChassisSpeeds` for the rotation of the robot to minimize drift.

0.02 is from the constant `kDt`, so I updated that.